pyxcp/master/base.py

```diff
@@ -170,6 +173,20 @@ def connect(self):
         self.DWORD_pack = makeDWordPacker(byteOrderPrefix)
         self.WORD_unpack = makeWordUnpacker(byteOrderPrefix)
         self.DWORD_unpack = makeDWordUnpacker(byteOrderPrefix)
+
+        if self.slaveProperties.addressGranularity == \
+                types.AddressGranularity.BYTE:
+            self.AG_pack = struct.Struct("<B").pack
+            self.AG_unpack = struct.Struct("<B").pack
```

Comment:
shouldn't this be struct.Struct("<B").**un**pack?

Reply:
Yup, good catch!
I'll fix it with a future pull request.